stix_shifter_modules/trustar/trustar.md

```diff
@@ -0,0 +1,291 @@
+# Trustar 
```

Comment:
This file should be renamed to README.md

Reply:
Done the changes.